apps/frontend/src/app/PageCharacter/CustomMultiTarget.tsx

```diff
@@ -70,14 +78,17 @@ function validateCustomTarget(ct: any): CustomTarget | undefined {
 
   return { weight, path, hitMode, reaction, infusionAura, bonusStats }
 }
-export function validateCustomMultiTarget(cmt: any): CustomMultiTarget | undefined {
-  let { name, targets } = cmt
+export function validateCustomMultiTarget(cmt: unknown): CustomMultiTarget | undefined {
+  if (typeof cmt !== "object") return undefined
+  let { name, description, targets } = cmt as CustomMultiTarget
   if (typeof name !== "string")
     name = "New Custom Target"
+  if (typeof description !== "string")
```

Comment:
a bit of defensive code against desc size would be nice, just in case someone is vindictive enough to add an exceptionally long desc. limit desc to like 2000 char